tests/general/RegexLintBearTest.py

```diff
@@ -0,0 +1,87 @@
+from os.path import abspath
+from queue import Queue
+
+from bears.general.RegexLintBear import RegexLintBear
+from coalib.testing.LocalBearTestHelper import (
+    LocalBearTestHelper, execute_bear)
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+test_good_python_file = """
+some_regex = r'[a-zA-Z]]'
+"""
+
+test_bad_python_file = """
+some_regex = r'(else|elseif)'
+"""
+
+test_good_cpp_file = """
+char some_regex[] = "[a-zA-Z]]";
+"""
+
+test_bad_cpp_file = """
+char some_regex[13] = "(else|elseif)";
+"""
+
+test_re_error_file = """
+some_regex = r'*ab' # This should be skipped
+some_other_regex = r'[a-z]'
```

Comment:
Why so many? We know it would be fine since we already tested it. Keeping this once is also fine...

Reply:
Yeah, I missed to delete them later :P